src/modules/noit_lua_noit_binding.c

```diff
@@ -69,6 +69,22 @@ nl_check(lua_State *L) {
 }
 
 static int
+nl_check_ud(lua_State *L) {
+  uuid_t *uuid;
+  uuid = lua_touserdata(L, 1);
+
+  noit_check_t *check;
+
+  check = noit_poller_lookup(*uuid);
```

Comment:
If we're not going to be strictly c89 compliant anyway, you can declare the check and init in the same line.. same with uuid above.